### ISSUE TITLE
Error when configured only one chain. [BUG] 

### ISSUE THREAD

**Issue by dylanngph** [family/connectkit#340]
**Describe the bug**

Issue happended when only configured one chain only, please help!

**To reproduce**

Follow the basic getting started and configure 1 chain only, for example chains: [mainnet] in the getDefaultConfig

**Comment by iainnash** [comment]
Ran into this issue where I'm asked to change chain to mainnet (same chain)

**Comment by lochie** [comment]
@dylanngph Do you have your `transports` set up for `[mainnet.id]`?
If you and @iainnash can share your configs and what versions of connectkit + wagmi + viem you're using that would help be able to debug this issue.

**Comment by lochie** [comment]
Interesting, I'll take a dive into this on Monday.

For now you can disable this Switch Networks popup by disabling `enforceSupportedChains`.
https://docs.family.co/connectkit/api-reference#enforceSupportedChains

Example:
```jsx
<ConnectKitProvider options={{
  enforceSupportedChains: false
}}>
```

**Comment by lochie** [comment]
This fix is now available in ConnectKit `1.7.1`. Thanks again for alerting us to this issue 🫡
